server/src/graql/executor/QueryExecutor.java

```diff
@@ -107,10 +109,17 @@ public QueryExecutor(TransactionOLTP transaction, boolean infer) {
             validateClause(matchClause);
 
             if (!infer) {
-                answerStream = matchClause.getPatterns().getDisjunctiveNormalForm().getPatterns().stream()
+
```

Comment:
Mark this code with a `TODO: <description>` so we remember to remove them in the future once we upgrade to Java 10, @flyingsilverfin